app/controllers/compute_resources_controller.rb

```diff
@@ -94,6 +94,7 @@ def test_connection
       @compute_resource = ComputeResource.new_provider(params[:compute_resource])
     end
     @compute_resource.test_connection :force => true
+    @success = true if @compute_resource.errors[:base].blank?
```

Comment:
What do you think about renaming the variable to `connection_success`? Also, setting this variable `=` to the result of `@compute_resource.test_connection(:force => true)` would be a bit less verbose as it will return true or false too.